Sources/ArcGISToolkit/Components/FeatureFormView/Subviews/FormElements/GroupFormElementView.swift

```diff
@@ -67,21 +71,15 @@ struct GroupFormElementView<Content>: View where Content: View {
 
 extension GroupFormElementView {
     /// A view displaying a label and description of a `GroupFormElement`.
-    struct Header: View {
+    struct Label: View {
```

Comment:
Since this view is only used in one place and the body is so simple, having it be a separate view seems a little overkill.